pallets/weights/src/pallet_utility.rs

```diff
@@ -0,0 +1,41 @@
+//! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
```

Comment:
Since this file has custom changes, please add a comment at the top that describes those changes. It will come in handy when we regenerate this file later.